analysers/analyser_merge_afigeo_hydrants.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Antonin Delpeuch 2020                                      ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, GeoJSON, Load, Mapping, Select, Generate
+
+
+class _Analyser_Merge_Afigeo_Hydrants(Analyser_Merge):
+    def __init__(self, config, source_url, dataset_name, source, osmRef, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+        # TODO: no idea how we pick the item number, and the id and level
+        self.def_class_missing_official(item = 8360, id = 1, level = 3, tags = ['merge', 'emergency'],
+            title = T_('Fire hydrant not integrated'))
+        self.def_class_possible_merge(item = 8361, id = 3, level = 3, tags = ['merge', 'emergency'],
+            title = T_('Fire hydrant integration suggestion'))
+        self.def_class_update_official(item = 8362, id = 4, level = 3, tags = ['merge', 'emergency'],
+            title = T_('Fire hydrant update'))
```

Comment:
We can reuse the item of Analyser_Merge_Hydrant_Point_CH_Lausanne, but since class id 1 and 3 was used, we use new id.
```suggestion
        self.def_class_missing_official(item = 8090, id = 11, level = 3, tags = ['merge', 'emergency'],
            title = T_('Fire hydrant not integrated'))
        self.def_class_possible_merge(item = 8091, id = 13, level = 3, tags = ['merge', 'emergency'],
            title = T_('Fire hydrant integration suggestion'))
        self.def_class_update_official(item = 8092, id = 14, level = 3, tags = ['merge', 'emergency'],
            title = T_('Fire hydrant update'))
```